packages/playground/src/components/deployment_data_dialog.vue

```diff
@@ -62,6 +62,7 @@
               <CopyReadonlyInput label="Network Name" :data="contract.interfaces[0].network" />
               <CopyReadonlyInput label="CPU (vCores)" :data="contract.capacity.cpu" />
               <CopyReadonlyInput label="Memory (MB)" :data="contract.capacity.memory" />
+              <CopyReadonlyInput label="SSD (GB)" :data="getStorage(contract.rootfs_size)" />
```

Comment:
That's only rootfs_size, there're zmounts (number of disks that are part of the deployment) 

https://github.com/threefoldtech/tfgrid-sdk-ts/blob/development/packages/grid_client/src/zos/workload.ts#L112